talent/job-descriptions/engineering-section.md

```diff
@@ -2,15 +2,17 @@
 
 Engineering consists of five different teams that represent the architecture of our product:
 
-- Applications (Scala, Go and Python and Frontend tools): Builds CLI/Web applications based on ML research.
-- Machine Learning (C++ and Python): R&D of machine learning on source code.
+- Applications (Scala, Go and Python and Frontend tools): Builds CLI/Web applications combining ML research with our stack.
+- Machine Learning (C++ and Python): R&D for machine learning on source code.
 - Data Retrieval (Scala and Go): Builds the technology that finds, fetches, stores and analyzes over +60M Git repositories.
-- Language Analysis (Go and another +15): Works on Babelfish, the universal code parsing server.
-- Infrastructure (Go and Python): Manages on-prem bare metal servers with Kubernetes and CoreOS.
+- Language Analysis (Go and another +15): focused on Babelfish, the universal code parsing server.
+- Infrastructure (Go and Python): Manages a cluser of on-prem bare metal servers with Kubernetes and CoreOS, and GCP for user facing applications. 
```

Comment:
add a link to GCP or use Google Cloud Platform (not everyone recognizes otherwise)